iosMath/render/MTMathUILabel.h

```diff
@@ -55,7 +61,11 @@ typedef NS_ENUM(unsigned int, MTTextAlignment) {
  When created it uses `[MTFontManager defaultFont]` as its font. This can be changed using
  the `font` parameter.
  */
+#if TARGET_OS_IPHONE
 IB_DESIGNABLE @interface MTMathUILabel : UIView
+#else
+IB_DESIGNABLE @interface MTMathUILabel : NSView
```

Comment:
We could do `NSView`, `UIView` similar to the color. Put them a header file defined to `MTView`.